january-slates/readme.md

```diff
@@ -1 +1,106 @@
-***Needs to be updated***
+[CLAMS.ai](https://clams.ai/) team  
+Author: Jeremy Huey, Data Annotation Manager @jarumihooi  
+Annotators: Unknown (GBH), Samantha Driscoll (GBH)
+Slates readme.md: version a.a - this version still requires gold generation information.   
+09/02/2023
+# Slates
+
+## Goal of the Project
+What is a film slate? - "The term film slate is still used to reference the clapper board that appears on the screen when filming. Just like early clappers. 
+The slate is shown at the beginning of a take immediately prior to the commencement of action." - [Beverly Boy Productions](https://beverlyboy.com/filmmaking/what-does-slate-mean-in-film/#:~:text=The%20term%20film%20slate%20is,the%20term%20slate%20in%20film.).  
+[More information about slates](https://www.youtube.com/watch?v=Heg6kDxXZ8k&t=13).  
+[What goes on a slate](theblackandblue.com/2012/11/05/deciphering-film-slate-1/).  
+This project creates a dataset that annotates where informational frames about the video are located timewise from many videos/GUIDs. 
+This dataset is needed to power automatic detection of slate information in the video collection at the AAPB. 
+Many of the collection pieces have incomplete or unverified information about what the video is. 
+This detected information could be used to verify and update metadata about that collection piece.
+
+### Project Information
+```
+Annotation project name - January Slates
+Annotator Demographics (2) -  
+Unknown (Unknown age, (other demographic information), Lang: Unknown, Organization: GBH)  
+Samantha Driscoll (Working Professional age, (other demographic information), Lang: Eng US native speaker, Organization: GBH)  
+Annotation Environment information -    
+    Name: TODO - Unknown  
+    Version: Unknown  
+    Link/Tool Used/User Manual: (See below Tool Installation)  
+Project changes -  
+    Project Version: Unknown/1.0  
+    Number of Batches: 1
+    Other Version Control Information: None
+```
+
+### Data Overview
+INPUT - a set of video files to be annotated. _TODO: Which tool?_   
+INTERMEDIATE RAW OUTPUT - `.csv` file where each line is the time of when the slate frames appear in that video. 
+```
+FORMAT:
```

Comment:
"FORMAT" being outside of code block?

Reply:
I was thinking something like this, with bullet points and indentations; 

---
### Data Overview
* INPUT - a set of video files to be annotated. _TODO: Which tool?_   
* INTERMEDIATE RAW OUTPUT - `.csv` file where each line is the time of when the slate frames appear in that video.
* Format
    * Fields: GUID, ..., Anything moving on screen during slate?
    * Example:
        ```
        cpb-aacip-81-881jx33t,",","Woman                     ,","00:00:00;00 ,","00:00:05;04 ,","handwriting  ,",recorded,,,",",boxes to fill in,",",no,
        cpb-aacip-41-34fn32g7,",","Carolina Journal          ,","00:00:00;00 ,","00:00:14;28 ,","typed        ,",digital?,,,",",key-value pairs,",",countdown,
        ```
        > [!NOTE]
        > Note the extra commas added in and extra blank columns and extra space within the column names. 

* FINAL PREPARED GOLD OUTPUT - `.tsv` file that seems to be actually comma-separated anyway. The gold files conform to the repository readme guideline that each gold must relate to only one GUID. Therefore, each of these gold files is only 1 video/GUID each. Note: the column headers are added to each file.  
    * FORMAT 
        * Fields: GUID, ..., Anything moving on screen during slate?
        * EXAMPLE:
            ```
            GUID,Series/Group,Slate Start,Slate End,Writing Types,Recorded/Digital,format of most of the information,Anything moving on screen during slate?
            cpb-aacip-29-01pg4g2x,Prime Time Wisconsin,00:00:00;00,00:00:08;14,typed,digital,key-value pairs,no
            ```

Reply:
Ok, I've changed it to match.